tests/build/run_cython_not_in_cwd.srctree

```diff
@@ -0,0 +1,56 @@
+
+PYTHON check_paths_from_cython.py
+
+######## src/foo/__init__.py ########
+
+# foo is a package and foo.bar is the module to build
```

Comment:
This becomes a useless comment if you write e.g. `pkg` instead of `foo` to give the test setup itself some semantics. `foo` and `bar` are semantically the same, but here, the package name and the module name are different things. Why not name them after what they are?